gtk4/src/subclass/widget.rs

```diff
@@ -1317,3 +1330,26 @@ where
 pub trait CompositeTemplate: WidgetImpl {
     fn bind_template(klass: &mut Self::Class);
 }
+
+pub type TemplateCallback = (&'static str, fn(&[glib::Value]) -> Option<glib::Value>);
+
+pub trait CompositeTemplateCallbacks {
+    const CALLBACKS: &'static [TemplateCallback];
+
+    fn bind_template_callbacks<T: WidgetClassSubclassExt>(klass: &mut T) {
+        Self::add_callbacks_to_scope(&klass.rust_template_scope());
+    }
+    fn bind_template_callbacks_prefixed<T: WidgetClassSubclassExt>(klass: &mut T, prefix: &str) {
+        Self::add_callbacks_to_scope_prefixed(&klass.rust_template_scope(), prefix);
+    }
```

Comment:
Is this actually useful?

Reply:
I plan to use it in one of my projects for some callbacks that are used from a different widget class, e.g. for a callback like this:

```rust
impl MyForm {
    #[template_callback]
    pub fn save(&self) {
        // ...
    }
}
```

In the widget's own template, that would be `<signal handler="save"/>`. If a container widget wanted to also use that function and avoid naming conflicts, it could import it with a prefix and then refer to it as `<signal handler="my_form_save"/>`.